contracts/forwarding/IForwarderV2.sol

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity ^0.4.24;
+
+import "./IForwarderPayable.sol";
+
+
+// TODO: Cannot inherit interfaces in Solidity 0.4.24
+contract IForwarderV2 is IForwarderPayable {
```

Comment:
I was wondering if we should instead call this `IForwarderRequiringContext` or etc. and base it off of `IForwarder`.

I don't think we should allow for the base `forward(bytes evmCallScript)` to be implemented in this case.

Reply:
Yea, definitely the name was to be discussed, proposing `IForwarderWithContext` now, let me know your thoughts!